laravel/app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use App\Http\Controllers\UserController;
+use App\Http\Controllers\TopController;
+use Illuminate\Http\Request;
+use App\Models\Post;
+use Auth;
+
+class PostController extends Controller
+{
+    public function index()
+    {
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     * 投稿内容を入力するページVIEW(post.blade.php)を返す
+     */
+    public function create(Request $request)
+    {
+      //ログイン状態確認。未ログインであれば会員登録ページへ飛ばす（この対処は仮）
+      $login_parameter = Auth::check();
+      if ($login_parameter === false) {
```

Comment:
一度しか使わない＆読みやすさに問題がなければ変数を使わないほうがすっきりします。

```suggestion
      if (Auth::check() === false) {
```